test/unit/firebase-app.spec.ts

```diff
@@ -433,7 +433,8 @@ describe('FirebaseApp', () => {
       return app2.delete();
     });
 
-    it('should throw when databaseURL is not set', () => {
+    it('should throw when databaseURL is not set, will fail if ' +
```

Comment:
Can we add a test case for the new functionality? Set the env variable and see app.database() works as intended.